cime_config/machines/config_machines.xml

```diff
@@ -600,12 +600,16 @@
       <executable>srun</executable>
       <arguments>
         <arg name="num_tasks"> -l -K -n {{ total_tasks }} -N {{ num_nodes }} </arg>
-        <arg name="binding">--cpu_bind=cores</arg>
+        <!-- <arg name="binding">cpu_bind=cores</arg>  -->
```

Comment:
@grnydawn Are you planning on updating the MAX_TASKS_PER_NODE to allow for more OMP threads? If so, can you check if process binding is okay with these flags in that scenario (MPI+OpenMP)?

Reply:
@sarats I will try to update the MAX_TASKS_PER_NODE and will also check the process binding.

Reply:
@sarats Changing MAX_TASKS_PER_NODE from 64 to 128 did request only one node instead of previous two nodes. In addition, I also had to add -c 2 flag for "salloc" job allocation to prevent resource error. Now, I will investigate further how to apply these settings to E3SM configuration files.